cmd/minikube/cmd/start.go

```diff
@@ -142,6 +142,8 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error setting up kubeconfig: ", err)
 		cmdUtil.MaybeReportErrorAndExit(err)
 	}
+	//Check here?
+	// util.CheckForKubectl()
```

Comment:
Might be redundant?  I think that if its running in the root command prerun it will always be triggered above.